.github/workflows/docker.yml

```diff
@@ -3,53 +3,13 @@ name: Build, test and push Docker Images
 
 on:
     pull_request:
-        paths:
-            - ".github/workflows/docker.yml"
```

Comment:
This seemed to include essentially all files and seemed very fragile. For now I simply excluded Markdown files, although I tempted to even remove that.

Reply:
Makes a lot sense for this repo, go ahead.